ethcore/src/state.rs

```diff
@@ -339,6 +339,18 @@ impl fmt::Debug for State {
 	}
 }
 
+impl Clone for State {
```

Comment:
in block.rs, `ClosedBlock` and also in miner.rs. it's necessary so that we can reopen a `ClosedBlock` later, add a new transaction and possibly seal.